### ISSUE TITLE
Tray Icon: Show waiting time for initial sync

### ISSUE THREAD

**Issue by moscicki** [owncloud/client#1614]
Initial sync of an existing large directory on the server (to the empty local folder) first seems to traverse the entire tree with PROPFIND. To the user the sync client seems to be stuck in this phase for a long time (depending on the size of the tree) before the sync starts. Unfortunately during this preparation process the status icons do not have a clear feedback what happens (sometimes the sync icon in the Account tab disappears).

In longer term, is it possible to avoid the full PROPFIND scan initially?

In short term, is it possible to make sure that the visual feedback while the initial scan gives a better impression to the users.

**Comment by dragotin** [comment]
This is done, we have the visual feedback during discovery phase now.
